TypeCobol/Compiler/Sql/CodeElements/SqlCodeElementCheckers.cs

```diff
@@ -16,10 +19,29 @@ public static class SavepointStatementChecker
                 if (savePointName.StartsWith("SYS", StringComparison.OrdinalIgnoreCase))
                 {
                     DiagnosticUtils.AddError(savepointStatement,
-                        "Invalid savepoint-name, it must not begin with 'SYS'.",
-                        context);
+                        "Invalid savepoint-name, it must not begin with 'SYS'.", context);
                 }
             }
         }
     }
+
+    internal static class AlterSequenceStatementChecker
+    {
+        public static void OnCodeElement(AlterSequenceStatement alterSequenceStatement,
+            List<IParseTree> duplicatedClauses, bool emptyClauseSet,
+            CodeElementsParser.AlterSequenceStatementContext context)
+        {
+            foreach (var clauseType in duplicatedClauses)
+            {
+                DiagnosticUtils.AddError(alterSequenceStatement,
+                    clauseType.GetChild(0).GetText() + " can not be specified more than once",
```

Comment:
`clauseType.GetChild(0).GetText()` won't work when NO clause is passed.
I think we need to change the grammar, keeping the noClauses branch but adding named variants for each

```
noClauses: noMinvalueClause | noMaxvalueClause | noOrderClause | noCycleClause | noCacheClause;
noMinvalueClause: SQL_NO minValue;
[...]
```

This allow to capture the whole clause in a single context object that can be passed to SetOption. The NO keyword will still be used for syntax properties and the keyword following it will be used in the diagnostic message.